hl13/code/game/machinery/computer/workforce_intake_terminal.dm

```diff
@@ -12,7 +12,8 @@
 	var/coupon_recorded_name
 	var/id_trim
 	var/coupon_age
-
+	var/coupon_ration_unit
```

Comment:
do we really need for this to have coupon ration unit? doesnt everyone starts at 0 anyway?

Reply:
Nope, they don't start at zero